Q/Qt6Base/build_tarballs.jl

```diff
@@ -134,10 +140,16 @@ dependencies = [
     BuildDependency(PackageSpec(name="LLVM_full_jll", version=v"11.0.1")),
 ]
 
+if !host_build
+    push!(dependencies, HostBuildDependency("Qt6Base_jll"))
+end
+
 include("../../fancy_toys.jl")
 
-if any(should_build_platform.(triplet.(platforms_macos)))
-    build_tarballs(ARGS, name, version, sources, script, platforms_macos, products_macos, dependencies; preferred_gcc_version = v"8", julia_compat="1.6")
+@static if !host_build
```

Comment:
For the record, a top-level `@static if` does nothing more than a simple `if`

Reply:
I got `platforms_macos` undefined with a regular if.